atomic_reactor/schemas/config.json

```diff
@@ -456,23 +456,38 @@
         }
     },
     "registries_organization": {"$ref": "#/definitions/organization"},
+    "registries_cfg_path": {
```

Comment:
now that we have only one registry I guess `registry_cgf_path` will be a better name.

Reply:
There is a discussion in the linked jira issue for this name.

It's not only for one registry. Multiple registries can be configured within that configuration file.